### PR TITLE
Update the edgecontainercluster integration test

### PR DESCRIPTION
**Pull request by rayccz2**
### Change description

Fix the edgecontainercluster integration test, local test passed: https://screenshot.googleplex.com/3F6y4h3aXr2HtF6.png

Tested with make ready-pr

- [x ] Run `make ready-pr` to ensure this PR is ready for review.
- [x ] Perform necessary E2E testing for changed resources.

### CODE REVIEW

**Comment by justinsb** [comment]
Test failure was an envtest flake, timeout shutting down kube-apiserver.  Triggered a retest.

**Comment by gemmahou** [comment]
Please update the [LONG_RUNNING_CRUD_TESTS_REGEX](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/scripts/shared-vars-public.sh#L103C1-L103C30) to use edgecontainercluster instead of edgecontainer.

You may also want to add the test regex here: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/pkg/test/constants/constants.go#L39 with a brief explanation

**Comment by gemmahou** [comment]
> Test failure was an envtest flake, timeout shutting down kube-apiserver. Triggered a retest.

This test needs ~2500s to finish, we'll need to configure it in [LONG_RUNNING_CRUD_TESTS_REGEX](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/scripts/shared-vars-public.sh#L103C1-L103C30). Also, the dependency project needs to have gkehub api enabled.

**Comment by rayccz2** [comment]
> Please update the [LONG_RUNNING_CRUD_TESTS_REGEX](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/scripts/shared-vars-public.sh#L103C1-L103C30) to use edgecontainercluster instead of edgecontainer.
> 
> You may also want to add the test regex here to disable nodepool and vpnConnection tests: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/pkg/test/constants/constants.go#L39 with a brief explanation



> Please update the [LONG_RUNNING_CRUD_TESTS_REGEX](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/scripts/shared-vars-public.sh#L103C1-L103C30) to use edgecontainercluster instead of edgecontainer.
> 
> You may also want to add the test regex here to disable nodepool and vpnConnection tests: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/pkg/test/constants/constants.go#L39 with a brief explanation

Done.

File: pkg/test/resourcefixture/testdata/basic/edgecontainer/v1beta1/edgecontainercluster/dependencies.yaml
Comment (gemmahou) [inline pkg/test/resourcefixture/testdata/basic/edgecontainer/v1beta1/edgecontainercluster/dependencies.yaml:1]: Could you update the other two dependencies.yaml for nodepool and vpnConnection?

Comment (rayccz2) [reply pkg/test/resourcefixture/testdata/basic/edgecontainer/v1beta1/edgecontainercluster/dependencies.yaml:1]: Done.

**Review by gemmahou — APPROVED** [review]

**Pull request merged**